sensor_msgs/PointCloud2[] pcl_arr
